### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [todotxt/todo.txt#82] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by waldyrious** [comment, marked as answer]
See #80.

**Reply by ghost** [reply]
Hi waldyrious, thank you for feedback.

**Comment by sycam0r-e** [comment]
New here, stumbled across todo.txt, directly fell in love. Would be amazing to see this being maintained (more), just wanted to express my hope and gratefulness for this project! :) Please keep going, it is so beautiful! :)
